lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -357,9 +357,7 @@
             <?php endif; ?>
 
             <div class="container__row">
```

Comment:
There shouldn't be a `container__row` nested inside a `container__block` (see line 346 above). I think that's what's messing up alignment here.